meta-refkit-core/recipes-ostree/refkit-ostree/refkit-ostree_0.0.bb

```diff
@@ -0,0 +1,42 @@
+SUMMARY = "IoT RefKit ostree helper, scripts, services, et al."
+
+LICENSE = "BSD-3-Clause"
+LIC_FILES_CHKSUM = "file://LICENSE-BSD;md5=f9f435c1bd3a753365e799edf375fc42"
+
+SRC_URI = " \
+  file://${PN} \
+"
+
+DEPENDS = "ostree"
+
+inherit autotools pkgconfig systemd distro_features_check
+
+REQUIRED_DISTRO_FEATURES = "ostree systemd"
+
+S = "${WORKDIR}/${PN}"
+
+PACKAGES += "${PN}-initramfs"
+
+FILES_${PN}-initramfs = " \
+    ${bindir}/refkit-ostree \
+"
+
+FILES_${PN} = " \
+    ${bindir}/refkit-ostree-update \
```

Comment:
I think it's better to do it now rather than later... that way we can make sure it gets properly done and not forgotten/omitted becuase of some other hurries. So let's do it now... I'll split out the most useful bits from that commit message and add it to doc/system-update.rst and update the PR once done.